cmd/kubelet/app/options/options.go

```diff
@@ -510,4 +511,5 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	fs.BoolVar(&c.RegisterNode, "register-node", c.RegisterNode, "Register the node with the apiserver. If --kubeconfig is not provided, this flag is irrelevant, as the Kubelet won't have an apiserver to register with.")
 
 	fs.Var(&utilflag.RegisterWithTaintsVar{Value: &c.RegisterWithTaints}, "register-with-taints", "Register the node with the given list of taints (comma separated \"<key>=<value>:<effect>\"). No-op if register-node is false.")
+	fs.IntVar(&dns.DNSDefaultNdots, "dns-default-ndtos", 5, "default dns ndtos setting for all pods in the node")
```

Comment:
```suggestion
	fs.IntVar(&dns.DNSNdots, "dns-ndtos", 5, "dns ndtos setting for all pods in the node")
```

Reply:
Better add one in https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/ than a new flag?